### DISCUSSION TITLE
[5.x]: Option to Hide Sidebar in Matrix Fields by Default and Adjust Panel Width on Smaller Screens

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#16033] [Ideas]
### What happened?

I have two improvement suggestions to enhance usability and space management in the Craft CMS editing panel, especially when working with Matrix fields:

## Hide the Sidebar with Status Information by Default or Provide a Setting

When working with Matrix fields or nested entries within Matrix fields, the sidebar displaying status information (e.g., entry status, publish options) takes up considerable space. Since this information is often not immediately relevant for editing, it would be helpful to have the sidebar hidden by default. Alternatively, a setting could be introduced to control the sidebar visibility specifically for Matrix fields—either globally via a PHP config file or as a user setting for individual customization. This would allow for a more focused editing experience, particularly when working with complex Matrix fields.

## Adjust Panel Width on Smaller Screens

To improve editing on mobile devices or smaller screens, it would be ideal if the editing panel could take up around 80–90% of the screen width when the screen size falls below 920 pixels (or in mobile view). This would provide the editor with additional space to comfortably adjust content on smaller devices.

I believe these adjustments would significantly improve the Craft CMS editing experience, especially when handling complex field structures like Matrix fields and when editing on smaller screens.

![Image](https://github.com/user-attachments/assets/673b6e6e-0884-44ba-a37e-91a2b41eed5d)
![Image](https://github.com/user-attachments/assets/9db576b7-859b-4fa2-a09f-96da0138a0f4)

### Craft CMS version

5.4

### Installed plugins and versions

-

**Comment by jurriaanr** [comment]
I was also looking for an option to do this, or possibly an event that could be intercepted to prevent the sidebar from being shown expanded
